modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache

```diff
@@ -3,6 +3,9 @@
 
 export const BASE_PATH = "{{{basePath}}}".replace(/\/+$/, "");
 
+const Blob = Blob !== undefined ? Blob : function() {};
```

Comment:
[CI](https://travis-ci.org/OpenAPITools/openapi-generator/builds/536245806?utm_source=github_status&utm_medium=notification) fails due to `runtime.ts(17,14): error TS2448: Block-scoped variable 'Blob' used before its declaration.`
In https://www.typescriptlang.org/play/index.html you can try
```typescript
const Blob = Blob !== undefined ? Blob : function () { };

function isBl(a: any) {
    return a instanceof Blob;
}
```
which does not work.
However, this works:
```typescript
if (typeof Blob === 'undefined') {
    const Blob = function () { };
}

function isBl(a: any) {
    return a instanceof Blob;
}
```

Reply:
Hey, seeing `isBl` gave me an idea - we could make a function that checks if the argument is a Blob and also returns false when Blob is undefined. I think it would be cleaner than conditionally defining consts.

Reply:
sounds good